utils/S3UtilsMongoClient.js

```diff
@@ -377,32 +400,42 @@ class S3UtilsMongoClient extends MongoClientInterface {
         };
     }
 
-    updateBucketCapacityInfo(bucketName, capacityInfo, log, cb) {
-        const m = this.getCollection(METASTORE);
-        m.findOneAndUpdate({
-            _id: bucketName,
-        }, {
-            $set: {
-                '_id': bucketName,
-                'value.capabilities.VeeamSOSApi.CapacityInfo': {
-                    Capacity: capacityInfo.Capacity,
-                    Available: capacityInfo.Available,
-                    Used: capacityInfo.Used,
-                    LastModified: (new Date()).toISOString(),
+    async updateBucketCapacityInfo(bucketName, capacityInfo, log, cb) {
```

Comment:
is this function is now `async`, may be cleaner to remove the `cb` and update caller sites with `.then(next).catch(next)`...

Reply:
It will be definitely cleaner, but this is not the role of this PR, switching from callbacks to promises will require to change almost all our implementation...